### ISSUE TITLE
Problem with tests

### ISSUE THREAD

**Issue by frenzymadness** [y-crdt/ypy-websocket#62]
I have a problem running tests and it seems that the problem is in this fixture:

https://github.com/y-crdt/ypy-websocket/blob/b5a7dbfb8516450a62ac83737554f00742cb58c0/tests/conftest.py#L26-L34

The error is:
```
______________________________ test_ypy_yjs_1[1] _______________________________

yws_server = <async_generator object yws_server at 0x7f2dccd6db70>
yjs_client = <Popen: returncode: None args: ['node', 'tests/yjs_client_1.js']>

    @pytest.mark.asyncio
    @pytest.mark.parametrize("yjs_client", "1", indirect=True)
    async def test_ypy_yjs_1(yws_server, yjs_client):
        # wait for the JS client to connect
        tt, dt = 0, 0.1
        while True:
            await asyncio.sleep(dt)
>           if "/my-roomname" in yws_server.rooms:
E           AttributeError: 'async_generator' object has no attribute 'rooms'
```

From the error, it seems that the fixture produces a standard generator instead of the instance of the server.

I googled a little bit and it seems that this fixture should be marked with `@pytest_asyncio.fixture` instead of `@pytest.fixture`. Source: https://stackoverflow.com/questions/72996818/attributeerror-in-pytest-with-asyncio-after-include-code-in-fixtures

**Comment by bnavigator** [comment]
The tests also passes if you use `pytest --asyncio-mode auto`: https://pytest-asyncio.readthedocs.io/en/latest/concepts.html#test-discovery-modes

**Comment by davidbrochart** [comment]
Fixed in #64.
